maintainers/maintainer-list.nix

```diff
@@ -1268,6 +1268,12 @@
     githubId = 17880;
     name = "Bodil Stokke";
   };
+  bootstrap-prime = {
```

Comment:
Not that it bothers me that much, but usually we try to keep names in lexicographical order. Please move your entry further down the list.